src/components/icons/clear-all-icon.js

```diff
@@ -1,4 +1,6 @@
-function clearAllIcon() {
-	return "<i class='icon fa-solid fa-c'></i>";
+import React from "react";
+
+export default function iconClearAll() {
```

Comment:
Use components files like

`AsteriskIcon.js`


change this for all components

Reply:
but using Icon {name} it's a pattern of react-icons

Reply:
aah ok

Reply:
done